### PR TITLE
#146 - creating-try-it-out-section

### PR DESCRIPTION
**Pull request by rohinij15**

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -59,3 +59,6 @@ main {
 footer {
   text-align: center;
 }
+.tryText {
```
Comment (mikelbrierly) [inline client/src/App.css:62]: Generally it's good practice to keep CSS classnames in "kebab case" like `.try-text`. And don't forget to change the class attribute in the HTML as well to match! :+1:

File: client/src/App.js [hunk 2]
```diff
@@ -109,7 +109,7 @@ function App() {
                  src="https://user-images.githubusercontent.com/13723156/127782418-a009e7a2-294d-4abe-b46e-2ac0baac5e40.png" 
                  alt="Circle logo"  
                />
-               <LinearGradient height="200%" /> 
+               <LinearGradient height="200%" width= "4px" />
```
Comment (mikelbrierly) [inline client/src/App.js:112]: I think linear gradient should automatically apply width if height is set, and vice versa. Can you test out that this component works fine still when you drop the `width` property?

File: client/src/App.js [hunk 3]
```diff
@@ -128,6 +128,12 @@ function App() {
            </div>
          </div>
         </section>
+        {/*try-it-out section */}
+        <section className="flex justify-evenly items-center mb-20 mt-28">
```
Comment (mikelbrierly) [inline client/src/App.js:132]: This is so great!! Fantastic use of flex and percentages!!!

File: client/src/App.js [hunk 4]
```diff
@@ -128,6 +128,12 @@ function App() {
...
+          <LinearGradient width="13%" height="4px" />
+          <p className="tryText">Here, why don't you just try it out?</p>
+          <LinearGradient width="13%" height="4px"/>
```
Comment (mikelbrierly) [inline client/src/App.js:135]: I think we can drop the `height` attribute here as well

File: client/src/App.js (hunk 2 above)
Comment (rohinij15) [reply client/src/App.js:112]: Yep I checked and it worked. However when we drop the width/height property depending on the situation, the line becomes thinner; that is the only difference

**Review by mikelbrierly — APPROVED** [review]

**Review by mikelbrierly — APPROVED** [review]

**Pull request merged**